Assets/MixedRealityToolkit-Preview/CameraCapture/CameraCaptureScreen.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.Collections;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.CameraCapture
+{
+	public class CameraCaptureScreen : ICameraCapture
+	{
+		Camera           sourceCamera;
```

Comment:
I find this often will dramatically improve the readability of code, so I do it almost everywhere! This kind of formatting is super important to me, so I'm gonna keep it unless someone has stong feeling otherwise :)